FirebaseAuth/Sources/SystemService/FIRAuthStoredUserManager.m

```diff
@@ -50,7 +50,7 @@ - (instancetype)initWithServiceName:(NSString *)serviceName {
 #pragma mark - User Access Group
 
 - (NSString *_Nullable)getStoredUserAccessGroup {
-  NSData *data = [self.userDefaults dataForKey:kStoredUserAccessGroupKey error:NULL];
+  NSData *data = [[self.userDefaults dataForKey:kStoredUserAccessGroupKey error:NULL] data];
```

Comment:
Fixes warning that causes runtime crash. <img width="1021" alt="Screen Shot 2023-04-10 at 9 53 01 PM" src="https://user-images.githubusercontent.com/36927374/231035104-2dd666e5-6b37-4a3b-b0b5-0fece261e6fc.png">